lhotse/bin/modes/recipes/broadcast_news.py

```diff
@@ -0,0 +1,27 @@
+import click
+
+from lhotse.bin.modes import prepare
+from lhotse.recipes.broadcast_news import prepare_broadcast_news
+from lhotse.utils import Pathlike
+
+
+@prepare.command()
+@click.argument('audio_dir', type=click.Path(exists=True, file_okay=False))
+@click.argument('transcript_dir', type=click.Path(exists=True, file_okay=False))
+@click.argument('output_dir', type=click.Path())
+def broadcast_news(
+        audio_dir: Pathlike,
+        transcript_dir: Pathlike,
+        output_dir: Pathlike
+):
+    """
+    English Broadcast News 1997 data preparation.
+    It will output three manifests: for recordings, topic sections, and speech segments.
+    It supports the following LDC distributions:
```

Comment:
We should probably note that these are not free unless your institution has an LDC subsription.
Eventually noobs will use this repo.